qemu/tests/cfg/slof_hugepage.cfg

```diff
@@ -0,0 +1,48 @@
+- slof_hugepage: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    type = slof_hugepage
+    restart_vm = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    start_vm = no
+    slots_mem = 4
+    mem = 2048
+    maxmem_mem = 32G
+    pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+    pre_command_noncritical = yes
+    free_mem_cmd = free -b | grep -E 'Mem' | awk '{print $2}'
+    only ppc64le ppc64
+    only Linux
+    # Select one drive format since just test slof with hugepage.
+    only virtio_scsi
+    variants:
+        - with_backing_file_hotplug:
+            # The unit of total_hugepage_size is 'MB'
+            total_hugepage_size = 4096
+            extra_params += " -mem-path /mnt/kvm_hugepage"
+            mem_devs = mem1
+            backend_mem1 = memory-backend-file
+            size_mem1 = 1G
+            policy_mem1 = default
+            mem-path_mem1 = /mnt/kvm_hugepage
+            plug_mem_name = plug
+            size_plug = 1G
+            policy_plug = bind
+            mem-path_plug = /mnt/kvm_hugepage
+            backend_plug = memory-backend-file
+            host-nodes_plug = 0
+            plug_timeout = 5
+        - without_backing_file_hotplug:
+            total_hugepage_size = 8192
+            mem_devs = mem1
+            backend_mem1 = memory-backend-file
+            size_mem1 = 2G
+            policy_mem1 = default
+            mem-path_mem1 = /mnt/kvm_hugepage
+            plug_mem_name = plug
+            size_plug = 2G
+            policy_plug = bind
+            mem-path_plug = /mnt/kvm_hugepage
+            backend_plug = memory-backend-file
+            host-nodes_plug = 0
+            plug_timeout = 5
```

Comment:
common params can be moved out of variants.

Reply:
Thanks for your comment, have updated.